spec/support/shared_sites_controller_shared_behavior.rb

```diff
@@ -26,7 +20,7 @@
     before { UserSession.create(users(:affiliate_manager_with_pending_approval_status)) }
 
     it 'should redirect to affiliates page' do
-      self.send request_method, action, args
+      self.send request_method, action, params: parameters
```

Comment:
Let's remove the `self` in `self.send` for consistency with the other specs.

Reply:
👍